### PR TITLE
Fix CLN channel balance reading

### PR DESCRIPTION
**Pull request by zynos**
Fix the problem that only the balance of the first channel is read.

### CODE REVIEW

File: get_CLN_data.sh [hunk 1]
```diff
@@ -45,7 +45,7 @@ ln_channelbalance=0
 #check if len(channels) == 0 --> no channels
 amount_of_channels=$(echo ${lncli_listfunds} | jq -r '.channels | length')
 if [ $amount_of_channels -gt 0 ];then
-ln_channelbalance=$(echo ${lncli_listfunds} | jq -r '.channels[0].our_amount_msat // 0 | gsub("msat";"") | tonumber' | awk '{print $1/1000}')
+ln_channelbalance=$(echo ${lncli_listfunds} | jq -r '[.channels[].channel_sat] | add')
```
Comment (vicariousdrama) [inline get_CLN_data.sh:48]: I don't have CLN installed (yet), but is the value of `our_amount_msat` the same as `channel_sat`? Or does one equal local balance and the other the capacity?

Comment (zynos) [reply get_CLN_data.sh:48]: Its the sat value without the "msat" at the end.

```
admin@raspibolt:~$ lightning-cli listfunds
"channels": [
      {
         "peer_id": "02XXXXXXXXXXXXXXXXXXXXXXXXx",
         "connected": true,
         "state": "CHANNELD_NORMAL",
         "short_channel_id": "7YYYYYYYYYY",
         "channel_sat": 50042,
         "our_amount_msat": "50042000msat",
         "channel_total_sat": 100000,
         "amount_msat": "100000000msat",
         "funding_txid": "ZZZZZZZZZZZZZZZZZZZZZZZZZZ",
         "funding_output": 0
      },

```

**Review by vicariousdrama — APPROVED** [review]
utAck

**Review by ghost — APPROVED** [review]
utACK

**Pull request merged**